.ci/pytorch/test.sh

```diff
@@ -329,6 +328,7 @@ test_inductor_distributed() {
   python test/run_test.py -i distributed/_composable/fsdp/test_fully_shard_mixed_precision.py -k test_reduce_dtype --verbose
   python test/run_test.py -i distributed/_composable/fsdp/test_fully_shard_clip_grad_norm_.py -k test_clip_grad_norm_2d --verbose
   python test/run_test.py -i distributed/fsdp/test_fsdp_tp_integration.py -k test_fsdp_tp_integration --verbose
+  python test/run_test.py -i distributed/_composable/test_composability/test_2d_composability.py -k test_train_parity_2d_mlp --verbose
```

Comment:
curious: why is this moved?

also, is `test_2d_composability` meant to be both in `test_inductor_distributed` (`test.sh`) and `multigpu-test.sh`?

Reply:
will recover it